docs/framework/release-notes/2025/01-28-january-cumulative-update-preview.md

```diff
@@ -0,0 +1,45 @@
+---
+title: January 2025 cumulative update preview
+description: Learn about the improvements in the .NET Framework January 2025 cumulative update preview.
+ms.date: 01/28/2025
+---
+# January 2025 cumulative update preview
+
+_Released January 28, 2025_
+
+## Summary of what's new in this release
+
+- [Security improvements](#security-improvements)
+- [Quality and reliability improvements](#quality-and-reliability-improvements)
+
+### Security improvements
+
+There are no new security improvements in this release. This update is cumulative and contains all previously released security improvements.
+
+### Quality and reliability improvements
+
+#### CLR
+
+Addresses an issue with printing large number of pages due to an OutOfMemory exception. To apply this improvement, you must set the following AppContextSwitch. (*Applies to: .NET Framework 4.8, 4.8.1*)
+
+<AppContextSwitchOverrides value="Switch.System.Windows.Controls.ReleaseDiscardableFontReasources=true"/>
```

Comment:
you need to put this in a code block for it to render properly (the triple tick block)

Reply:
gotcha, done :)